src/content/docs/en/pages/guides/marketplace/templates/turso-with-drizzle.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: How to deploy the Turso Starter Kit with Drizzle template
+description: This template contains the configurations to connect an Edge Application to a Turso database.
+meta_tags: Drizzle ORM, Turso database, Edge Function, Azion Marketplace, web, guides, templates
+namespace: docs_guides_turso_with_drizzle
+permalink: /documentation/products/guides/turso-with-drizzle/
+---
+
+import Tag from 'primevue/tag';
+
+<Tag severity="info" client:only="vue">
+Preview
+</Tag>
+
+The **Turso Starter Kit with Drizzle ORM** template contains the configurations for creating and connecting an edge application to a Turso database via Drizzle ORM.
+
+The deployment of this template creates a GitHub repository containing your project, as well as an edge application and a domain to facilitate your access and management through Azion Edge Platform. It also creates a new database with a new table in your Turso account.
+
+## Requirements
+
+- A [GitHub account](https://github.com/signup) to connect with Azion and create your new repository.
+    - Every push will be deployed automatically to this repository to keep your project updated.
+- A [Turso account](https://turso.tech/) with an empty database slot.
+- This template uses [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), and [Edge Cache](https://www.azion.com/en/documentation/products/build/edge-application/edge-cache/), and it could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
+
+:::note
+Database storage and operations could also generate costs directly by Turso. Check [Turso's pricing page](https://turso.tech/pricing) for details.
+:::
```

Comment:
```suggestion

This template uses [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), and [Edge Cache](https://www.azion.com/en/documentation/products/build/edge-application/edge-cache/), and it could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
Database storage and operations could also generate costs directly by Turso. Check [Turso's pricing page](https://turso.tech/pricing) for details.
```